frontend/src/utils/AuthContext.tsx

```diff
@@ -100,24 +100,29 @@ export const AuthProvider = ({ children }: AuthProviderProps) => {
     "/login",
     "/signup",
     "/password/forgot",
-    "/password/reset/*",
+    "/password/reset/",
```

Comment:
why remove asteriks? I thought the asteriks is supposed to match all the links that have /password/reset?